privacy-config/privacy-config-impl/src/main/java/com/duckduckgo/privacy/config/impl/features/trackingparameters/TrackingParametersPlugin.kt

```diff
@@ -56,7 +58,8 @@ class TrackingParametersPlugin @Inject constructor(
             }
 
             trackingParametersRepository.updateAll(exceptions, parameters)
-            val isEnabled = trackingParametersFeature?.state == "enabled"
+            val isEnabled = trackingParametersFeature?.state == "enabled" &&
+                versionHandler.isSupportedVersion(trackingParametersFeature.minSupportedVersion)
```

Comment:
We should also add this to all the other plugins. It probably doesn't matter right now but the vpn module also contains its own remote config plugins so we probably need to find a way to be able to share it there as well... I can think of a different options but maybe moving to the public API of remote config is a nice way to do it? cc/ @aitorvs